sql/updates/0.0.1--0.0.2.sql

```diff
@@ -1,18 +1,27 @@
 -- here for reference of directory file structure
-DROP OPERATOR CLASS IF EXISTS vector_l2_ops using embedding CASCADE;
+DROP OPERATOR CLASS IF EXISTS vector_l2_ops
+    USING embedding CASCADE;
+
 DROP ACCESS METHOD IF EXISTS embedding;
+
 DROP FUNCTION IF EXISTS hnsw_handler;
 
-CREATE FUNCTION hnsw_handler(internal) RETURNS index_am_handler
-	AS 'MODULE_PATHNAME' LANGUAGE C;
+CREATE FUNCTION hnsw_handler (internal)
+    RETURNS index_am_handler
+    AS 'MODULE_PATHNAME'
```

Comment:
I have a weak preference of not touching old update files. These are supposed to be immutable after the corresponding release. Formatting does not change the logic so not a big deal